tutorials/monitoring/tutorial.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Monitoring NServiceBus solutions
+reviewed: 2017-10-02
+summary: Getting started with the monitoring capabilities built in to the Particular Platform.
+---
+
+This tutorial will guide you through getting started with the monitoring capabilities built in to the Particular Platform.
+
+You can follow along with this tutorial to configure any solution for monitoring with the Particular Platform. If you do not have a solution to monitor just yet, you can get started by downloading our sample solution.
+
+downloadbutton
+
+
+## Component overview
+
+The following diagram shows all of the components that are involved in monitoring an NServiceBus solution.
+
+```mermaid
+graph TD
+
+subgraph Endpoint1
+  metrics1(NServiceBus.Metrics) --> plugin1
+  plugin1(NServiceBus.Metrics.ServiceControl)
+end
+
+subgraph Endpoint2
+  metrics2(NServiceBus.Metrics) --> plugin2
+  plugin2(NServiceBus.Metrics.ServiceControl) 
+end
+
+plugin1 --> q(Monitoring Queue)
+plugin2 --> q(Monitoring Queue)
+
+q --> scmonitoring[Monitoring Instance]
+
+scmonitoring --> pulse[ServicePulse]
+```
+
+![the above renders like this](component-overview.png)
+
+Each endpoint in the solution is configured to collect metrics about it's own operation using the NServiceBus.Metrics NuGet package. This package hooks into the NServiceBus pipeline and calculates key metric data. It also adds headers to outgoing messages to track queue length.
```

Comment:
> ...and calculates key metric data.

This sentence sounds stunted. It feels like it should have "... using X", "...to Y", or "...which is sent to Z" or something similar appended to it.

Reply:
It's a weird situation because the package doesn't do anything with the data, it just makes it available. I changed it to: 

> This package hooks into the NServiceBus pipeline to calculate and expose key metric data.